.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and tests
```

Comment:
could you please rename this file to `run-checks-tests.yaml`? I am now working on the release that will automatically check the github actions builds, and the filename is included in the API URL so it would be handy have the name unified across all our packages :blush:

Reply:
renamed here: https://github.com/shopsys/JsFormValidatorBundle/pull/1/commits/c2965b26eef3111a12e9cf66052167947f6c9e47